css/styles.css

```diff
@@ -155,6 +155,69 @@ ul {
     text-decoration: underline;
 }
 
+/* Dropdown */
+
+.dropdown-menu {
+    position: absolute;
+    width: 300px;
+    height: 400px;
+    border: 1px solid var(--Alto);
+    border-radius: 5px;
+    right: 30px;
+    margin-top: 10px;
+    box-shadow: 0 0 0.3em var(--Alto);
+    background-color: var(--White);
+    overflow-y: auto;
+}
+
+.dropdown-menu__list {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
```

Comment:
List items should be aligned to the left (please compare it with the _"design"_)